tensorboard/webapp/metrics/store/metrics_selectors_test.ts

```diff
@@ -348,7 +348,7 @@ describe('metrics selectors', () => {
     });
   });
 
-  describe('getCardTimeSeries', () => {
+  describe('getLoadableTimeSeries', () => {
```

Comment:
The tests in this is still using getCardTimeSeries selector. Could you make another test group (another describe) for getLoadableTimeSeries?